src/RoomInfoListItem.cc

```diff
@@ -77,12 +77,12 @@ RoomInfoListItem::RoomInfoListItem(RoomInfo info, QWidget *parent)
 	avatarLayout_->addWidget(roomAvatar_);
 
 	roomName_ = new QLabel(info_.name(), textWidget_);
-	roomName_->setMaximumSize(parent->width() - 10, 20);
+	roomName_->setMaximumSize(parent->width() - max_height_, 20);
```

Comment:
Yes you're right. But on Linux the situation is a little different. The room topic is cut off a lot sooner.

![room-list](https://cloud.githubusercontent.com/assets/6430350/25564645/6f613cd0-2dc0-11e7-8259-c28ad45c9982.png)


The problem with the room name is that the font size isn't taken into consideration. Same goes for the room topic. We should use `QFontMetrics` to cut off the text. Maybe the screen resolution has an impact on this and this was the problem all along.

![test](https://cloud.githubusercontent.com/assets/6430350/25564633/26f259de-2dc0-11e7-821b-a8c277f330e0.png)